Gemfile

```diff
@@ -23,8 +23,8 @@ end
 
 gem 'dhasher'
 gem 'discard', :git => 'https://github.com/jhawthorn/discard', :branch => 'master'
+gem 'insights-api-common', '~> 3.0'
```

Comment:
Should we go back to using master for this so we don't have to wait on new releases in the common gem?

Reply:
@lindgrenj6 For now we are in sync with the latest version so we dont have to be at master for now. As and when we need it we can change the Gemfile to reflect that.